From 9b46b865883a8f43a8fa3ea3044aa49e59fd7b2c Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Wed, 12 Mar 2025 21:06:31 +0100
Subject: [PATCH] nemea-siot: add missing library in ConfigParser.h

Added missing library <cstdint> in ConfigParser.h for fixing compilation error
with missing uint64_t type
---
 net/nemea-siot/patches/010-add-missing-library.patch | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 net/nemea-siot/patches/010-add-missing-library.patch

diff --git a/net/nemea-siot/patches/010-add-missing-library.patch b/net/nemea-siot/patches/010-add-missing-library.patch
new file mode 100644
index 0000000..b034ffb
--- /dev/null
+++ b/net/nemea-siot/patches/010-add-missing-library.patch
@@ -0,0 +1,12 @@
+Index: nemea-siot-19-08.1/wsn-anomaly/ConfigParser.h
+===================================================================
+--- nemea-siot-19-08.1.orig/wsn-anomaly/ConfigParser.h
++++ nemea-siot-19-08.1/wsn-anomaly/ConfigParser.h
+@@ -45,6 +45,7 @@
+ #include <fstream>
+ #include <algorithm>
+ #include <sstream>
++#include <cstdint>
+ #include "Conversion.h"
+ 
+ using namespace std;
-- 
2.48.1

